test/libsolidity/natspecJSON/docstring_enum_value.sol

```diff
@@ -0,0 +1,25 @@
+contract C {
+    enum Color {
+        Red,
+        /// @title example of title
+        /// @author example of author
+        /// @notice example of notice
+        /// @dev example of dev
+        Green
+    }
+}
+// ----
+// ----
+// :C devdoc
+// {
+//     "kind": "dev",
+//     "methods": {},
+//     "version": 1
+// }
+//
+// :C userdoc
+// {
+//     "kind": "user",
+//     "methods": {},
+//     "version": 1
+// }
```

Comment:
The test is the same as the other one?

Reply:
Looks like this is also the case for `docstring_enum.sol`/`enum_no_docs.sol` these seem to be based on.

Apparently both were added in #13626, but `docstring_enum.sol` was originally a syntax test. Since Natspec JSON tests are a superset of syntax tests (they do full analysis and IIRC can also display errors the way syntax tests do), when I created it, I moved some Natspec-related syntax tests there (dc68480f72cb7321be3e228877ceb70ed7678f92). So I could have just removed it, because it is simply redundant.

We should do that now. Let's leave just one of each and call them `enum.sol` and `enum_value.sol`.